libs/langchain-anthropic/src/chat_models.ts

```diff
@@ -367,11 +344,6 @@ export interface AnthropicInput {
    */
   invocationKwargs?: Kwargs;
 
-  /**
-   * Whether or not to include token usage data in streamed chunks.
-   * @default true
-   */
-  streamUsage?: boolean;
```

Comment:
Deleting this would be a breaking change, we need this + only streaming usage metadata if this is set to true.